src/component/screen.js

```diff
@@ -1,26 +1,42 @@
-const Screen = () => (
-  <div className="calculator-content">
-    <p className="calc-item item-one">0</p>
-    <div className="calc-item">0</div>
-    <div className="calc-item">AC</div>
-    <div className="calc-item">+/-</div>
-    <div className="calc-item operator">/</div>
-    <div className="calc-item">7</div>
-    <div className="calc-item">8</div>
-    <div className="calc-item">9</div>
-    <div className="calc-item operator">*</div>
-    <div className="calc-item">4</div>
-    <div className="calc-item">5</div>
-    <div className="calc-item">6</div>
-    <div className="calc-item operator">-</div>
-    <div className="calc-item">1</div>
-    <div className="calc-item">2</div>
-    <div className="calc-item">3</div>
-    <div className="calc-item operator">+</div>
-    <div className="calc-item item-zero">0</div>
-    <div className="calc-item">.</div>
-    <div className="calc-item operator">=</div>
-  </div>
-);
+import { useState } from 'react';
+import calculate from '../logic/calculate';
+
+const Screen = () => {
+  const [calcul, setCalcul] = useState({ total: 0, next: null, operation: null });
+  const handleClick = (e) => {
+    const data = e.target.textContent;
+    const calc = calculate(calcul, data);
+    setCalcul(calc);
+  };
+  const { total, next, operation } = calcul;
+  return (
+    <div className="calculator-content">
+      <p className="calc-item item-one">{ next || operation || total || 0}</p>
+      <button type="button" className="calc-item" onClick={handleClick}>0</button>
+      <button type="button" className="calc-item" onClick={handleClick}>AC</button>
+      <button type="button" className="calc-item" onClick={handleClick}>+/-</button>
```

Comment:
### Your Calculator
![image](https://user-images.githubusercontent.com/93039359/228777412-f7a604b9-ae97-4ba9-922c-dc362e5618f0.png)

### Design Calculator
![image](https://user-images.githubusercontent.com/93039359/228776934-960cfb87-e215-48fc-a7b9-491ac4cb3c9a.png)

- [ ] Kindly rearrange the buttons of your calculator to make your calculator look similar to the design calculator to make your project professional and according to the requirements 👍